index.js

```diff
@@ -3,8 +3,12 @@
 var path = require('path');
 var isglob = require('is-glob');
 var pathDirname = require('path-dirname');
+var isWin32 = require('os').platform() === 'win32';
 
 module.exports = function globParent(str) {
+	// flip windows path separators
+	if (isWin32 && str.indexOf('/') < 0) str = str.split('\\').join('/');
```

Comment:
Yes, that's pretty much my assumption. Why extract the glob parent if you're then going to try to treat it as a glob again? The purpose imo is to have a string you can pass to `fs` methods.

And yes, re-joining after escape chars have been removed would be a mistake. That's not the intended use, and I don't see why someone would find it necessary to do so.